js/src/components/google-ads-account-card/google-ads-account-card.js

```diff
@@ -30,6 +39,20 @@ export default function GoogleAdsAccountCard() {
 		return <NonConnected />;
 	}
 
+	// Ads account has been created but we don't have access yet.
+	if (
```

Comment:
That's weird. When I tested, the component showed up whenever `has_access` was `false`. Can you give it another try?
We cannot make the modal as part of the `CreateAccountButton` component since it will stop rendering since the `NonConnected` component won't be rendered when you have an account created.